src/main/java/net/atos/zac/app/taken/TakenRESTService.java

```diff
@@ -85,11 +85,12 @@
 @Produces(MediaType.APPLICATION_JSON)
 public class TakenRESTService {
 
-    @Inject
-    private TakenService takenService;
+    public static final String TAAK_DATA_ZAAK_HERVATTEN = "zaakHervatten";
```

Comment:
Verplaats de TAAK_DATA_* variabelen naar TaakVariabelenService

Reply:
Ik heb ze allemaal in de code opgezocht en verhuist naar de TaakVariabelenService